src/locate/app.py

```diff
@@ -0,0 +1,16 @@
+from flask import Flask
+import logging
+import os, sys
+from flask_cors import CORS
+from flask_pymongo import PyMongo
+from controllers.locate import locate_blueprint
```

Comment:
same case here....we are seeing changes for the netmanager product yet the PR is for the location registry miscroservice. 🤔